vsphere/datadog_checks/vsphere/vsphere.py

```diff
@@ -930,7 +935,12 @@ def submit_property_metrics(
 
     def check(self, _):
         # type: (Any) -> None
-        self._hostname = datadog_agent.get_hostname()
+        if self._config.hostname_to_upper:
+            self._hostname = datadog_agent.get_hostname().upper()
+        elif self._config.hostname_to_lower:
+            self._hostname = datadog_agent.get_hostname().lower()
+        else:
+            self._hostname = datadog_agent.get_hostname()
```

Comment:
```suggestion
        self._hostname = datadog_agent.get_hostname().upper()
```
we don't need to modify the agent hostname since we get it from the agent, not vSphere

Reply:
Just went with the default here--`self._hostname = datadog_agent.get_hostname()`